frontend/src/router/router.js

```diff
@@ -3,8 +3,8 @@ import { useUserStore } from '@/stores/user';
 
 const routes = [
   {
-    path: '/',
-    name: 'home',
+    path: '/home2',
+    name: 'home2',
```

Comment:
Love the name 😄

Reply:
Thanks, the naming wizard in me really popped off here 😂 This code will be removed in the future cleanup PR. 😁